Marlin/src/inc/Conditionals_adv.h

```diff
@@ -543,3 +543,7 @@
 #if EITHER(MEATPACK_ON_SERIAL_PORT_1, MEATPACK_ON_SERIAL_PORT_2)
   #define HAS_MEATPACK 1
 #endif
+
+#if DISABLED(DISABLE_DRIVER_SAFE_POWER_PROTECT) && PIN_EXISTS(SAFE_POWER)
```

Comment:
I don't think `PIN_EXISTS(SAFE_POWER)` is ever evaluated here either since a single `#if PIN_EXISTS(SAFE_POWER) #error "safe power pin found"` check doesn't throw an error.

Reply:
It seems that the `PIN_EXISTS` macro can't be called in `Conditionals_adv.h`.  So I moved this judgment to `Conditionals_post.h` file in https://github.com/thisiskeithb/Marlin/pull/6